nat-lab/tests/test_events.py

```diff
@@ -18,6 +26,41 @@
 from utils.router import IPStack
 
 
+def node_cmp(left: TelioNode, right: TelioNode):
```

Comment:
nit: It sounds like this could be part of the TelioNode class or maybe a fn from /utils?

Reply:
`TelioNode` comes from the generated bindings so we unfortunately can't add it there, and I did consider adding it to `utils/bindings.py` but it's only used in this file so I decided to put it here. If you think it's still a better idea to move it to `utils/bindings.py` I can do that, I don't have any strong opinions on the matter

Reply:
fair enough, let's leave it here